snippet-extractor-output/snippets.yaml

```diff
@@ -316,7 +316,7 @@ Excel.CustomXmlPartCollection.set:
         targetRange.format.autofitColumns();
         await context.sync();
     });
-Excel.CustomXmlPart.add:
+Excel.CustomXmlPartCollection.add:
```

Comment:
Should this target CustomXMLPart.getXml instead?

Reply:
The same snippet is already/also tagged for CustomXMLPart.getXml